README.md

```diff
@@ -0,0 +1,22 @@
+#Project ToolShop
+
+User can buy tools for repairing
+
+#How install a project?
+
+1.Install VS Code
+2.npm init playwright@latest  --> install playwright latest version
```

Comment:
nodeJS needs to be installed first :)
Then we need to perform `npm install` - it will install all the dependencies needed.

npm init playwright@latest - need to be laucnhed only once, during first setup